.github/actions/spelling/excludes.txt

```diff
@@ -1,3 +1,4 @@
+(?:(?i)\.png$)
```

Comment:
Order doesn't matter in this file, but, I'm sorting it (and the bot will try to as well).